Assets/MixedRealityToolkit/_Core/Boundary/MixedRealityBoundarySystem.cs

```diff
@@ -66,35 +71,137 @@ public override void Reset()
             InitializeInternal();
         }
 
+        public bool Contains(Vector3 location)
+        {
+            return Contains(location, Boundary.Type.TrackedArea);
```

Comment:
Why not just make boundaryType = tacked area by default instead of having this overload?

Reply:
same pattern as the june18_dev branch. i can make the parameter have a default value